src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -18,65 +18,75 @@ To better understand this page, we recommend you first read our [introduction to
+
+Blockchains have a form of captcha of their own. This process of preventing fake identities from taking over a network is called "Sybil resistance." Like captcha or premium accounts on a website, Sybil resistance mechanisms associate some kind of work or value to participating in the blockchain. This causes an innate investment in the success of the network.
+
+**Sybil resistance** describes how a protocol fares against a [Sybil attack](https://en.wikipedia.org/wiki/Sybil_attack). Sybil attacks are when one user or group pretends to be many users. Resistance to this type of attack is essential for a decentralized blockchain and enables miners and validators to be rewarded equally based on resources put in. Proof-of-work and proof-of-stake protect against this by making users expend a lot of energy or put up a lot of collateral. These protections are an economic deterrent to Sybil attacks.
+
+Proof-of-work and proof-of-stake are not consensus protocols by themselves, but since each blockchain needs some form of Sybil resistance, it's easy to conflate the two concepts. They actually are Sybil resistance mechanisms and block author selectors. They are a way to decide who is the author of the latest block. It's this Sybil resistance mechanism combined with proposal mechanism that makes up a truly decentralized consensus protocol.
+
+### Proposal mechanism {#proposal-mechanism}
 
-Now technically, proof-of-work and proof-of-stake are not consensus protocols by themselves, but they are often referred to as such for simplicity. They are actually Sybil resistance mechanisms and block author selectors; they are a way to decide who is the author of the latest block. It's this Sybil resistance mechanism combined with a chain selection rule that makes up a true consensus mechanism.
+A proposal mechanism is what's used to determine the next block proposed on the network. In Nakamoto consensus, a person who mines a winning nonce gets the right to be the leader for the next block. They propose that block to the network. In the new proof-of-stake system, a random beacon decides the order of proposers from the list of available validators. In both cases, after a block is proposed, it's up to the next proposer to decide if they want to build on that block or on another block proposed in the case of competing proposals. This is what's known as the "chain selection rule."
 
-**Sybil resistance** measures how a protocol fares against a [Sybil attack](https://wikipedia.org/wiki/Sybil_attack). Sybil attacks are when one user or group pretends to be many users. Resistance to this type of attack is essential for a decentralized blockchain and enables miners and validators to be rewarded equally based on resources put in. Proof-of-work and proof-of-stake protect against this by making users expend a lot of energy or put up a lot of collateral. These protections are an economic deterrent to Sybil attacks.
+### Chain selection rule {#chain-selection-rule}
```

Comment:
Kind of splitting hairs here but it would be more consistent with other Ethereum literature and pages on ethereum.org to refer to "fork choice algorithm" or "fork choice rule" rather than "chain selection rule".

Also see previous comment regardng LMD-GHOST as the fork-choice algorithm on the Beacon Chain.